CefGlue.CopyLocal.props

```
@@ -3,6 +3,7 @@
     <ItemGroup Condition="'$(Platform)' == 'x64'">
         <PackageReference Include="chromiumembeddedframework.runtime.win-x64" />
         <PackageReference Include="cef.redist.osx64" />
+        <PackageReference Include="cef.redist.linux64" />
```

we should have another props file just with this section, so that we dont have to change 2 places
```suggestion
        <PackageReference Include="cef.redist.linux-x64" />
```